### DISCUSSION TITLE
How to initilize linter in Angular?

### DISCUSSION THREAD

**Discussion by Maliksncf** [angular-eslint/angular-eslint#1673] [Q&A]
Hello,

I work in a project using Angular 17 and Spring boot and i have a ticket from the sprint board to initialize linter in the front, i followed the github by adding wthe schematics and the packages in package.json and in angular.json however for the .eslintrc.json there is so many and different way i dont know which rule to use and which example in the github to choose can you help me to get through?

Thanks

**Comment by JamesHenry** [comment]
The `.eslintrc.json` that the schematics generated for you is the recommended config to start with. Anything else you wish to add on top is subjective and down to personal preference.

**Reply by Maliksncf** [reply]
thanks for your time and your answer
